lib/methsel.g

```diff
@@ -213,7 +213,15 @@ end;
 
 #F  # # # # # # # # # # #  verbose try next method  # # # # # # # # # # # # #
 ##
-
+NEXT_VMETHOD_PRINT_INFO := function ( methods, i, arity)
+    Print("#I Trying next: ", methods[(arity+4)*(i-1)+(arity+4)]);
+    if FILENAME_FUNC(methods[(arity+4)*(i-1)+(arity+2)]) <> fail then
+        Print(" at ",
+              FILENAME_FUNC(methods[(arity+4)*(i-1)+(arity+2)]), ":",
+              STARTLINE_FUNC(methods[(arity+4)*(i-1)+(arity+2)]));
+    fi;
+    Print("\n");
+end;
```

Comment:
Reading `methods[(arity+4)*(i-1)+(arity+2)]` three times in a row like this makes my eyes bleed :-P. Could you perhaps put it into a local variable? That would make the code easier to understand, too.